### ISSUE TITLE
Ensure a focus style is provided for main header logo

### ISSUE THREAD

**Issue by keelanfh** [localgovdrupal/localgov_base#376]
### Impact

Keyboard-only users require a clear focus indicator style for all interactive elements. This ensures they can track where their focus is on any given page and help ensure they can navigate easily. 

### WCAG Fail 

This item is a Fail of the WCAG 2.1 AA guidelines

### Expected results

User moves focus to the linked logo in the main header and sees a clear focus indicator. This indicator should have a minimum of a 3:1 contrast ratio with its adjacent colours. 

### Suggested fix

Check to ensure the linked element (<a>) which wraps the logo image has an authored focus style in the website stylesheets and no other issues prevent the user from seeing this. 

### Steps to reproduce

Starting in the browser address bar - e.g. at https://demo.localgovdrupal.org, press Tab key until you see the ‘Skip to content’ link appear, press once more and notice there is no focus indicator on the website logo (continue to menu links for comparison)

**Comment by markconroy** [comment]
You know what's strange about this? There's no focus style in Chrome, but there is in Firefox. I'm going to get a fix for it now.

**Comment by markconroy** [comment]
PR Ready
